Formula/qcachegrind.rb

```diff
@@ -12,16 +11,16 @@ class Qcachegrind < Formula
     sha256 "d16d0cd82b7f7e69862a636c70fb72aaddb450fdaea8653a77486e321ef811b6" => :yosemite
   end
 
-  depends_on "graphviz" => :optional
   depends_on "qt"
+  depends_on "graphviz" => :optional
 
   def install
-    cd "qcachegrind"
-    system "#{Formula["qt"].bin}/qmake", "-spec", "macx-clang", "-config", "release"
-    system "make"
-    # Install app
-    prefix.install "qcachegrind.app"
-    # Symlink in the command-line version
-    bin.install_symlink prefix/"qcachegrind.app/Contents/MacOS/qcachegrind"
+    cd "qcachegrind" do
```

Comment:
This has always been an open-ended question nobody gave enough of a crap about to mandate a rule on either way 😄. Loosely what I used to enforce was that if it didn't make indentation completely stupid `cd` should be passed with `do` but otherwise a simple `cd` was fine. 

I have since delegated any desires for Domness onto you, so, whatever you're happy with 😸.